snekbox/memfs.py

```diff
@@ -135,12 +138,17 @@ def files(
             exclude_files: A dict of Paths and last modified times.
                 Files will be excluded if their last modified time
                 is equal to the provided value.
+            timeout: The maximum time for the file parsing. If exceeded,
+                a TimeoutError will be raised.
         """
-        count = 0
-        for file in self.output.rglob(pattern):
-            # Ignore hidden directories or files
-            if any(part.startswith(".") for part in file.parts):
-                log.info(f"Skipping hidden path {file!s}")
+        start_time = time.monotonic()
+        added = 0
```

Comment:
What's the thought behind renaming this to `added`? The previous name, `count`, was clearer to me. Nothing is being "added" (at least not as far as this function is concerned).

Reply:
Yeah I think I'll change this back to count, previously I added then removed a `scanned` variable to stop at some determined depth hence leading to the name change.

Reply:
Changed back to `count` in [e924217](https://github.com/python-discord/snekbox/pull/173/commits/e9242175ced6584084b6a4a1c77d0319b49e2cc0)